packages/neuron-ui/src/widgets/Table/table.module.scss

```diff
@@ -17,6 +17,9 @@ $head-height: 52px;
     }
   }
 }
+.noBorder {
+  border: none !important;
```

Comment:
Try to avoid using "important" as much as possible. Using this keyword often implies that there are issues with our code design, and we need to carefully consider a reasonable solution.
Each "important" can become a ticking time bomb of technical debt.